api/forgot.go

```diff
@@ -54,11 +57,19 @@ func (a *Api) passwordReset(res http.ResponseWriter, req *http.Request, vars map
 		return
 	}
 
-	resetCnf, _ := models.NewConfirmation(models.TypePasswordReset, models.TemplateNamePasswordReset, "")
-	resetCnf.Email = email
-
 	// if the reseter is already a Tidepool user, we can use his preferences
-	if resetUsr := a.findExistingUser(resetCnf.Email, a.sl.TokenProvide()); resetUsr != nil {
+	if resetUsr := a.findExistingUser(email, a.sl.TokenProvide()); resetUsr != nil {
+		if resetUsr.IsClinic() || a.Config.AllowPatientResetPassword {
```

Comment:
should it not be an "AND" here? It has to be a clinician and the feature is enabled so the email for regular resetting password is sent. So I think the whole section needs to be reviewed because it would be a problem in the "else" section if we put a "AND" here

Reply:
The "OR" is fine here. 
We enter in the block to send standard email whenever the `a.Config.AllowPatientResetPassword` is set to `true`

Reply:
right